.gitignore

```diff
@@ -31,3 +31,6 @@ coverage.xml
 requirements.in
 requirements.txt
 .eggs/
+
+# OS
+.DS_Store
```

Comment:
I'm using macOS and I think it's annoying that git can see these files.

Reply:
I use this:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
With this https://github.com/github/gitignore/blob/master/Global/macOS.gitignore

Reply:
I second this. Global git ignore helps.

Reply:
Ok I see, I will revert this.
Thanks for the links @graingert!